maven-extension/src/test/java/io/opentelemetry/maven/OpenTelemetrySdkServiceTest.java

```diff
@@ -5,42 +5,117 @@
 
 package io.opentelemetry.maven;
 
-import org.junit.jupiter.api.Disabled;
+import static io.opentelemetry.api.common.AttributeKey.stringKey;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.opentelemetry.sdk.autoconfigure.spi.ConfigProperties;
+import io.opentelemetry.sdk.resources.Resource;
 import org.junit.jupiter.api.Test;
 
 public class OpenTelemetrySdkServiceTest {
 
-  /** Verify default `service.name` */
+  /** Verify default config */
   @Test
-  @Disabled
   public void testDefaultConfiguration() {
-    testConfiguration("maven");
+    System.clearProperty("otel.exporter.otlp.endpoint");
+    System.clearProperty("otel.service.name");
+    System.clearProperty("otel.resource.attributes");
```

Comment:
[minor] As an alternative you could use JUnit `@ClearSystemProperty` and `@SetSystemProperty` as it helps to prevent global state side-effects and allows to preserve existing values (if any), see [docs](https://junit-pioneer.org/docs/system-properties/) for more details.

Reply:
Great idea.
JUnit Pioneer requires Java 11 ([here](https://github.com/junit-pioneer/junit-pioneer/blob/v2.3.0/build.gradle.kts#L30)) when we support Java 8.
I captured a TODO to simplify the code once otel-java-contrib bumps to Java 11.

Reply:
just fyi, we don't have any plans to bump the whole repo to Java 11, but we can bump an individual module to Java 11 if it's warranted, e.g.

https://github.com/open-telemetry/opentelemetry-java-contrib/blob/d1cc766de244c9737e1c5b3859b723e82003c078/static-instrumenter/agent-instrumenter/build.gradle.kts#L24-L26

Reply:
Thanks @trask , I updated the comment in the code indicating that there was no plan to bump to Java 11.